boa_inputgen/src/ident_walk.rs

```diff
@@ -0,0 +1,506 @@
+use boa_engine::syntax::ast::{
```

Comment:
I am not very into fuzzing, so I'm not fully sure on how this works. One option here would be to document all functions and so on (which would be nice), or to document it at the module level (or both, which would be even better). I like the approach, definitely, but it's missing some documentation so that we can understand how to maintain it.

Reply:
All good! Thanks :)